salt/kubernetes-minion/init.sls

```diff
@@ -62,13 +62,21 @@ kubelet:
     - require:
       - service: kubelet
 
-# note: br_netfilter is not available in some kernels
-#       not sure we really need it...
-#
-#      - kmod: br_netfilter
-#
-#br_netfilter:
-#  kmod.present
+  # TODO: This needs to wait for the node to register, which takes a few seconds.
+  # Salt doesn't seem to have a retry mechanism in the version were using, so I'm
+  # doing a horrible hack right now.
+  cmd.run:
+    - name: |
+        NEXT_WAIT_TIME=0
+        UNCORDON_RET=1
+
+        until [ $UNCORDON_RET -eq 0 ] || [ $NEXT_WAIT_TIME -eq 10 ]; do
+          kubectl --kubeconfig {{ pillar['paths']['kubeconfig'] }} uncordon {{ grains['fqdn'] }}
+          UNCORDON_RET=$?
+          sleep $(( NEXT_WAIT_TIME++ ))
+        done
+
+        exit $UNCORDON_RET
```

Comment:
Done + tested it works.. 

First attempt:

	"stderr": "Error from server (NotFound): nodes \"a1efe6040a1646b99188dc60dd52a526.infra.caasp.local\" not found", 

Second (and last) attempt:

	#"stdout": "node \"a1efe6040a1646b99188dc60dd52a526.infra.caasp.local\" already uncordoned"